rootfs_overlay/etc/usb_gadget/hidg.schema

```diff
@@ -0,0 +1,83 @@
+attrs :
```

Comment:
Does this file HAVE to be tabbed in so deeply?  I always cringe when I see tabs instead of spaces, but maybe that's just me... 😅

Reply:
:+1: Changing the whitespace worked. I made the indentation 2 space characters to match Elixir's formatting.